channels/sa.m3u

```diff
@@ -79,24 +71,12 @@ https://hiplayer.hibridcdn.net/t/rotana-lbc.m3u8
 http://makkahtv.srfms.com:1935/makkahtv/livestream/chunklist_w1150651953.m3u8
 #EXTINF:-1 tvg-id="MakkahTV.sa" tvg-name="Makkah TV" tvg-country="SA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/5p2yUBb.png" group-title="Religious",Makkah TV (480p)
 http://makkahtv.srfms.com:1935/makkahtv/livestream/playlist.m3u8
-#EXTINF:-1 tvg-id="MakkahTV.sa" tvg-name="Makkah TV" tvg-country="SA" tvg-language="" tvg-logo="" group-title="",Makkah TV
-https://5ab29cc78f681.streamlock.net/makkahtv/livestream/chunklist_w2097348824.m3u8?v=2957056943
-#EXTINF:-1 tvg-id="MakkahTV.sa" tvg-name="Makkah TV" tvg-country="SA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/5p2yUBb.png" group-title="Religious",Makkah TV (480p)
-https://5ab29cc78f681.streamlock.net/makkahtv/livestream/chunklist_w655950915.m3u8
 #EXTINF:-1 tvg-id="MakkahTV.sa" tvg-name="Makkah TV" tvg-country="SA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/5p2yUBb.png" group-title="Religious",Makkah TV (480p)
 https://5ab29cc78f681.streamlock.net/makkahtv/livestream/playlist.m3u8
-#EXTINF:-1 tvg-id="Mekkatv.sa" tvg-name="Mekka tv" tvg-country="SA" tvg-language="" tvg-logo="" group-title="",Mekka tv (480p)
-http://makkahtv.srfms.com:1935/makkahtv/livestream/playlist.m3u8?v=
+#EXTINF:-1 tvg-id="MakkahTV.sa" tvg-name="Makkah TV" tvg-country="SA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/5p2yUBb.png" group-title="Religious",Makkah TV (480p)
+http://makkahtv.srfms.com:1935/makkahtv/livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="PanoramaFM.sa" tvg-name="Panorama FM" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Music",Panorama FM (1080p)
 https://shls-panoramafm-prod-dub.shahid.net/out/v1/66262e420d824475aaae794dc2d69f14/index.m3u8
-#EXTINF:-1 tvg-id="QuranTV.sa" tvg-name="Quran TV" tvg-country="SA" tvg-language="" tvg-logo="" group-title="",Quran TV
-http://5b18be6964c2f.streamlock.net:1935/live/_definst_/quran/chunklist_w2066488187.m3u8
-#EXTINF:-1 tvg-id="QuranTV.sa" tvg-name="Quran TV" tvg-country="SA" tvg-language="" tvg-logo="" group-title="",Quran TV
-http://m.live.net.sa:1935/live/quran/chunklist_w232396531.m3u8?v=2957056943
-#EXTINF:-1 tvg-id="QuranTV.sa" tvg-name="Quran TV" tvg-country="SA" tvg-language="" tvg-logo="" group-title="",Quran TV
-http://m.live.net.sa:1935/live/quran/chunklist_w994500087.m3u8
-#EXTINF:-1 tvg-id="QuranTV.sa" tvg-name="Quran TV" tvg-country="SA" tvg-language="" tvg-logo="" group-title="",Quran TV (720p)
-http://m.live.net.sa:1935/live/quran/gmswf.m3u8
```

Comment:
works

Reply:
It's a duplicate of http://m.live.net.sa:1935/live/quran/playlist.m3u8